clang/lib/Basic/LangOptions.cpp

```diff
@@ -243,3 +244,46 @@ LLVM_DUMP_METHOD void FPOptionsOverride::dump() {
 #include "clang/Basic/FPOptions.def"
   llvm::errs() << "\n";
 }
+
+std::optional<uint32_t> LangOptions::getCPlusPlusLangStd() const {
+  if (!CPlusPlus)
+    return std::nullopt;
+
+  LangStandard::Kind Std;
+  if (CPlusPlus26)
```

Comment:
I was thinking we could also generate this mapping if we hooked up the LangOptions name to the LangStandard macros. My concern is: every time there's a new standard version, we need to update a *lot* of places; reducing that isn't a bad idea. However, that's also probably a more involved refactoring, so I don't insist (or could be done in a follow-up).